rigex-front-end/src/pages/Landing/Landing.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-import { Route } from 'react-router-dom';
+import { Route, useLocation, Redirect } from 'react-router-dom';
+import validatePath from '../../utils/validatePath';
 
 import routePaths from '../../constants/routePaths';
 import { Rig } from '../index';
 import NavBar from '../../components/navBar/NavBar'
 
 const Landing = () => {
+  const { pathname } = useLocation();
```

Comment:
I think you can keep all of these const in the util file you made and just call the function with no argument since none of these are used anywhere else

Reply:
moved what I could